tools/data/sthv1/preparing_sthv1.md

```diff
@@ -7,16 +7,60 @@ Before we start, please make sure that the directory is located at `$MMACTION2/t
 
 First of all, you have to sign in and download annotations to `$MMACTION2/data/sthv1/annotations` on the official [website](https://20bn.com/datasets/something-something/v1).
 
-## Step 2. Prepare Videos
+## Step 2. Prepare RGB Frames or Videos
 
-Then, you can download all data parts to `$MMACTION2/data/sthv1/` and use the following command to extract.
+Since the sthv1 website doesn't provide the original video data and only extracted RGB frames are available, here are two alternatives to choose:
+a. For users who only want to use RGB frames, you can directly download RGB frames from [sthv1 website](https://20bn.com/datasets/something-something/v1).
+b. For users who need videos and RGB + Flow, you should download the videos from [sthv2 website](https://20bn.com/datasets/something-something/v2) and go with the following steps to extract them.
```

Comment:
As i understand, this is preparing v1, right?

Reply:
Yes，since original video data are not available on sthv1 website, we have to extract two-stream data based on sthv2 videos
 and sthv1 annos.

Reply:
What do you mean by saying "have to extract two-stream data based on sthv2 videos"?

Reply:
E.g., this is the 1.webm from v2 and video no 1 frames from v1

![image](https://user-images.githubusercontent.com/9464825/87877046-9ff1d380-ca0e-11ea-95c5-5670b469a834.png)

Reply:
Oh, they are different.

Reply:
Since the video data of sthv1 are not easy to get, you can directly use the RGB frames to extract flow by using denseflow. It supports frame inputs.